english-helper/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from "react";
+
+export function useLocalStorage(initialState: unknown, key: string) {
```

Comment:
export function useLocalStorage<T>(initialState: T, key: string): [T, React.Dispatch<React.SetStateAction<T>>] {
  const [value, setValue] = useState<T>(() => {
    const storedValue = localStorage.getItem(key);
    return storedValue ? JSON.parse(storedValue) : initialState;
  });
  
  we can use this ts approach
  don't use unknown or any